### PR TITLE
Update chat options for 202207

### PR DESCRIPTION
**Pull request by BigLep**
Given the push to drive to docs.ipfs.io/community in https://github.com/ipfs/ipfs/issues/487, updated the chat information to be accurate as of 2022-07-08.

### CODE REVIEW

File: docs/community/chat.md [hunk 1]
```diff
@@ -1,19 +1,25 @@
...
 ---
 
 # Chat
 
+In general, public IPFS Discord channels are bridged to Matrix and Slack.  As a result, you can usually use your preferred chat application to engage with the community.  IPFS channels/rooms are usually prefixed with "#ipfs-".
+
 ## Discord
 
-Join the [IPFS Discord server](https://discord.com/invite/KKucsCpZmY) for synchronous chat!
+Join the [IPFS Discord server](https://discord.com/ipfs) for synchronous chat! Server documentation is [here](https://pl-strflt.notion.site/IPFS-Discord-Server-Documentation-85bbc451303a473bbf6846b01610e3c1).
```
Comment (johnnymatthews) [inline docs/community/chat.md:13]:
```suggestion
Join the [IPFS Discord server](https://discord.com/ipfs) for synchronous chat! [Server documentation can be found on Notion](https://pl-strflt.notion.site/IPFS-Discord-Server-Documentation-85bbc451303a473bbf6846b01610e3c1).
```

File: docs/community/chat.md [hunk 2]
```diff
@@ -1,19 +1,25 @@
...
 
 ## Matrix
 
-Join the [Matrix group](https://matrix.to/#/#ipfs:matrix.org)
+Join the [Matrix space](https://matrix.to/#/#ipfs-space:ipfs.io).  Homeserver documentation is [here](https://pl-strflt.notion.site/IPFS-Matrix-Homeserver-Documentation-31481e76843547b7ab5160a87eed2b9f).
```
Comment (johnnymatthews) [inline docs/community/chat.md:17]:
```suggestion
Join the [Matrix space](https://matrix.to/#/#ipfs-space:ipfs.io). [Homeserver documentation is available on Notion](https://pl-strflt.notion.site/IPFS-Matrix-Homeserver-Documentation-31481e76843547b7ab5160a87eed2b9f).
```

**Comment by filecorgi** [comment]
- Image optimization came back clean!
- Vuepress build was successful!

File: docs/community/chat.md [hunk 3]
```diff
@@ -1,19 +1,25 @@
...
+
+## Slack
+
+Join [Filecoin slack](https://filecoin.io/slack) and join the bridged #ipfs channels.  Filecoin Slack workspace userguide is [here](https://pl-strflt.notion.site/Filecoin-Slack-User-Guide-29072679986b4ccb8ad7b091097dd770).
```
Comment (johnnymatthews) [inline docs/community/chat.md:21]:
```suggestion
Join [Filecoin slack](https://filecoin.io/slack) and join the bridged #ipfs channels. The [Filecoin Slack workspace user guide is available on Notion](https://pl-strflt.notion.site/Filecoin-Slack-User-Guide-29072679986b4ccb8ad7b091097dd770).
```

**Review by johnnymatthews — APPROVED** [review]

**Comment by filecorgi** [comment]
- Image optimization came back clean!
- Vuepress build was successful!

**Review by lidel — APPROVED** [review]

**Pull request merged**